tabs+awesome.css

```diff
@@ -0,0 +1,156 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;700&family=Raleway:wght@100;400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400&display=swap');
+/*Стиль для AWESOME*/
+.awesome {
+  font-family: 'Raleway', sans-serif;
+  width: 1170px;
+  margin:auto ; 
+  }
+
+.awesome__title {
+    margin: 0;
+    font-size: 36px;
+    font-weight: 400;
+  }
+
+.awesome__title-wrapper {
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  }
+
+.awesome__line {
+  border-bottom: 1px solid #EC7979;
+  width: 170px;
+  margin: auto;
+  margin-bottom: 40px;
+ 
+}  
+
+.awesome__subtitle {
+margin: 0;
+color: #86878B;
+font-size: 18px;
+text-align: center;
+margin-bottom: 110px;
+}
+
+.awesome__container {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  }
+
+.awesome__block { 
+  width: 370px; 
+}
+
+.awesome__icon-wrapper {
+  height: 92px;
+  width: 92px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #EC7979;
+  border-radius: 50%;
+  margin: auto;
+  margin-bottom: 78px;
+}
+.awesome__block-title {
+ font-family: 'Open Sans', sans-serif;
+ margin-bottom: 47px;
+}
+
+/*Стиль для TABS*/
+
+.tabs {
+  margin: auto;
+  width: 1170px;
+  margin-bottom: 50px;
+}
+
+.tabs__info {
+  margin: auto;
+  height: 500px;
+  border: 1px solid #C6CBCD;
+  display: flex;
+  justify-content: space-around; 
+}
+
+.tabs__text {
+  margin-left: 100px;
+  margin-top: 47px;
+  margin-right: 30px;
+  font-family: 'Raleway', sans-serif;
+  font-size: 18px;
+  }
+
+.tabs__title {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  margin: 0;
+  margin-bottom: 74px;
+  }
+
+.tabs__button {
+  margin-top: 10px;
+  height: 58px;
+  width: 270px;
+  background: #EC7979;
+  color: #FFFFFF;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 600;
+  font-size: 18px;
+  align-items: center;
+  border:none;
+  cursor: pointer;
+}
+
+.tabs__button:hover {
+  background: #AB4949;
+}
+
+.tabs__button:active {
+  background: #EC7979;
+box-shadow: inset 0px 0px 10px #AB4949;
+}
+
+.tabs__switch {
+  display: flex;
+  justify-content:flex-start;
+  align-items: center;
+  font-size: 14px;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 400px; 
+}
+
+.tabs__switch-name {
+  width: 203px;
+  height: 40px;
+  margin: 0;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  background: #FFFFFF;
+}
+
+.tabs__switch-name:focus { 
+  border-radius: 5px 5px 0px 0px;
+  border: 1px solid #C6CBCD;
+  margin-bottom: -4.5px;
+  border-bottom: #fff;
+}
+.tabs__switch-name:hover {
+  border-radius: 5px 5px 0px 0px;
+  border: 1px solid #C6CBCD;
+  margin-bottom: -4.5px;
```

Comment:
Your tabs are jumping on hovering. It's not correct. Margin-bottom should be always and not only in hover state if you want to avoid jumping.